Data/models/PlaylistFolder.swift

```diff
@@ -83,6 +83,21 @@ final public class PlaylistFolder: NSManagedObject, CRUD, Identifiable {
         folder.delete()
     }
     
+    public static func updateFolder(folderID: NSManagedObjectID, _ update: @escaping (PlaylistFolder?, Error?) -> Void) {
```

Comment:
nit but no fix needed in this PR: Use `Result<PlaylistFolder, Error>` in these situations instead of two optionals